tests/integration/test_iterators.py

```diff
@@ -322,6 +322,7 @@ def _check_asserts(self, das: DistributedAtomSpace, iterator: CustomQuery):
             next(iterator)
         return handles
 
+    @pytest.mark.skip("Requires Mongo indices to be updated with the new custom attributes")
```

Comment:
Please make sure there's a card to address this fix and reference the card repo/number in the skip message.

Reply:
Probably I missed this detail in the tests that have been skipped in the atomdb PR https://github.com/singnet/das-atom-db/pull/248 . Please fix the messages there as well.